ng/src/web/components/chart/schedule.js

```diff
@@ -50,6 +52,15 @@ const margin = {
   left: 60,
 };
 
+const MAX_LABEL_LENGTH = 25;
+
+const tickFormat = val => {
+  if (val.toString().length > MAX_LABEL_LENGTH) {
```

Comment:
the check is not required. shorten only cuts the text if it is longer then the second param see https://github.com/greenbone/gsa/blob/master/ng/src/gmp/utils/string.js#L38